### ISSUE TITLE
Actor 3.0 server is not running!!

### ISSUE THREAD

**Issue by NadeshG** [actorapp/actor-platform#971]
It shows
Cannot acquire connection from data source
org.postgresql.util.PSQLException: FATAL: password authentication failed for user "xxx"
...
ERROR im.actor.server.db.DbExtension$ DbExtension$(akka://actor-server)   Migration failed

But it runs on the previous version successfully with the same configuration!!!!

**Comment by rockjam** [comment]
@NadeshG Thanks for report. What was previous version you used?

**Comment by NadeshG** [comment]
@rockjam 1.0.153 is the previous version.. I suspect in the current version it is not connecting to the postgresql database..

**Comment by rockjam** [comment]
@NadeshG Yes, you are right, error you posted in first comment is related to postgres database connection. I'll check it bit later. Did you install  server update via Ubuntu PPA?

**Comment by NadeshG** [comment]
@rockjam I didn't made the server update.. I installed it freshly in a new server

**Comment by rockjam** [comment]
Well, are you sure you have same postgres settings on new server? Same user, password and  access for postgres is configured right in pg_hba.conf?

**Comment by NadeshG** [comment]
I installed the actor server and run it manually, ( not as a service ) instead run as
bin/actor.. In fact I have two versions of actor servers.. When I run the 1.0.153 then it goes good.. but in 3.0.0 facing the above error.
Also, on another note while running the actor server 1.0.153  I frequently receiving the following error!! 

error while starting up loggers
akka.ConfigurationException: Logger specified in config can't be loaded [akka.event.slf4j.Slf4jLogger] due to [akka.event.Logging$LoggerInitializationException: Logger log1-Slf4jLogger did not respond with LoggerInitialized, sent instead [TIMEOUT]]

To resolve this I want to restart my instance... Not sure the reason.. May be the timout time has to be increased somewhere in configuration

**Comment by rockjam** [comment]
Okay, I will check manual start of server too. But we actually didn't change server config code from previous release, so I advice you to check your settings once again. Make sure you can connect to postgres database with credentials you provide in config file.

regarding the error: 
```
akka.ConfigurationException: Logger specified in config can't be loaded [akka.event.slf4j.Slf4jLogger] due to [akka.event.Logging$LoggerInitializationException: Logger log1-Slf4jLogger did not respond with LoggerInitialized, sent instead [TIMEOUT]]
```

Try to put this line in your server.conf file:
```
akka.logger-startup-timeout = 50s
```
it should solve it

**Comment by rockjam** [comment]
@NadeshG Just checked last server version. It works fine both as a service and when launching it via `bin/actor`. You should dig more. Make sure your postgres configuration and credentials are correct. If they are - look for exceptions in server logs

**Comment by NadeshG** [comment]
@rockjam.. Thanks for the TIMEOUT issue in the 1.0.153 configuration... I will use it.. But for the latest server 3.0.0 there is issue in running the server.. I have checked for the config which connects well in 1.0.153 but not with 3.0.0

**Comment by rockjam** [comment]
It could be config issue, try to look into server logs more closely

**Comment by NadeshG** [comment]
Okay, Let me check on it.. BTW how to define 
akka.logger-startup-timeout = 50s  in the server.conf
May I simply add this line it the file ? or is there any format ?

**Comment by rockjam** [comment]
Just simply add this line to server.conf

**Comment by rockjam** [comment]
Not sure to which file your server logs are written. Check /var/log/actor/server.log first of all

**Comment by NadeshG** [comment]
Checked on the logs where I have received 
`Timeout failure stats (total=0, active=0, idle=0, waiting=0)
Exception in thread "main" org.flywaydb.core.api.FlywayException: Unable to obtain Jdbc connection from DataSource`
I have both version in a single instance now.. Starting the old version works good... but with the latest version above error. 
Does both in a single instance may be the issue ?

**Comment by rockjam** [comment]
Yes

**Comment by NadeshG** [comment]
Okay I will try to create a new separate instance for latest version and check on it
